cmd/login.go

```diff
@@ -103,6 +103,11 @@ func (lca loginCmdArgs) process() error {
 		return err
 	}
 
+	if lca.tenantID == "" || lca.tenantID == "common" {
+		glcm.Info("WARNING: Logging in under the \"Common\" tenant. This will log the account in under its' home tenant, NOT under your target tenant.")
```

Comment:
typos here:

glcm.Info("WARNING: Logging in under the \"Common\" tenant. This will log the account in under **its** home tenant, NOT under your target tenant.")
		glcm.Info("If you plan to use **AzCopy** for B2B **Storage Accounts**, please sign in under the target tenant with --tenant-id")

Reply:
Will fix in a moment, thanks for catching me.